### DISCUSSION TITLE
Color table of contents

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#2263] [Q&A] [answered]
Is there some way of getting the table of contents in, for instance, blue?

Thanks in advance!

**Comment by Dherse** [comment, marked as answer]
If you're talking about the color of the text, it's relatively easy using show rules:

```typ
#show outline: set text(fill: blue)
```

you can generally style it using show rules on `outline` or show rules on `outline.entry` :)

Relevant docs:
- https://typst.app/docs/tutorial/formatting/#show-rules
- https://typst.app/docs/reference/meta/outline/
- https://typst.app/docs/reference/meta/outline/#definitions-entry

**Reply by psads-git** [reply]
Thanks, @Dherse! That works! (I am just starting learning `typst`.)

**Reply by Dherse** [reply]
No worries, glad I could help, it took me a little while to figure out show rules, but once you do, they're very very powerful 🎉
